shotover-proxy/examples/cassandra-passthrough/docker-compose.yml

```diff
@@ -8,3 +8,7 @@ services:
       MAX_HEAP_SIZE: "128M"
       MIN_HEAP_SIZE: "128M"
       HEAP_NEWSIZE: "24M"
+      CASSANDRA_ENABLE_SCRIPTED_USER_DEFINED_FUNCTIONS: "true"
+      CASSANDRA_ENABLE_USER_DEFINED_FUNCTIONS: "true"
+    volumes:
+      - ./docker-entrypoint.sh:/usr/local/bin/docker-entrypoint.sh
```

Comment:
Cant we just specify the path here relative to the default CWD instead of changing the CWD?

Reply:
I frequently run the examples from within the directory and I would assume users will attempt the same. Also if you just download the example files to run them the paths won't be right.

Reply:
ah! good point

Reply:
Wait, uh... how does the redis-passthrough work?
It is already doing the same volume thing.

Reply:
...interesting. I thought the volume mounting worked relative to the processes CWD not the directory of the `docker-compose.yml`.

Reply:
Reverted the changes.